src/common/users.ts

```diff
@@ -406,7 +406,9 @@ export const checkAndClearUserStreak = async (
     return false;
   }
 
-  const today = getTodayTz(timezone);
+  const today = utcToZonedTime(new Date(), timezone);
```

Comment:
Does it really make any difference?

Reply:
I don't think it does, but it does seem more logical to use it instead of our own util.